hw/ip/pattgen/doc/dv_plan/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "PATTGEN DV Plan"
+---
+
+## Goals
+* **DV**
+  * Verify all PATTGEN IP features by running dynamic simulations with a SV/UVM based testbench
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+* **FPV**
+  * Verify TileLink device protocol compliance with an SVA based testbench
```

Comment:
@cindychip, this is set for most IPs. Do we have a path to track this as a part of nightly runs?

Reply:
Currently IP level (non-FPV) runs have fusesoc compile error because of the auto-generated RAL csr checks. I am trying to debug on that. The result will be here once fix the compile error: https://reports.opentitan.org/hw/top_earlgrey/fpv/summary.html